pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+prefer-workspace-packages: true
+packages:
+  - 'packages/**'
```

Comment:
You may want to set `packages/*` instead to make pnpm faster; otherwise it's going to attempt to recurse.